geoengine/error.py

```diff
@@ -19,10 +19,12 @@ class GeoEngineException(Exception):
     def __init__(self, response: Union[geoengine_openapi_client.ApiException, Dict[str, str]]) -> None:
         super().__init__()
 
-        if isinstance(response, geoengine_openapi_client.ApiException):
+        if isinstance(response, dict):
```

Comment:
why did you need to change this? What happened to the ApiException type in the openapi client?

Reply:
I don't remember, but I was able to revert this.

Reply:
[9235965](https://github.com/geo-engine/geoengine-python/pull/187/commits/923596557d6bf89ccf3a42014535a797cf0ff1a4)